### PR TITLE
[JBPM-9317] userTaskService.saveContent doesn't work

### PR DESCRIPTION
**Pull request by elguardian**
JIRA:
https://issues.redhat.com/browse/JBPM-9317

### CODE REVIEW

**Comment by elguardian** [comment]
Jenkins retest this please.

**Comment by elguardian** [comment]
depends on 
https://github.com/kiegroup/droolsjbpm-knowledge/pull/461
https://github.com/kiegroup/jbpm/pull/1747
https://github.com/kiegroup/droolsjbpm-integration/pull/2216

**Comment by elguardian** [comment]
Jenkins retest this please.

File: kie-internal/src/main/java/org/kie/internal/identity/IdentityProvider.java [hunk 1]
```diff
@@ -27,4 +27,12 @@
     List<String> getRoles();
 
     boolean hasRole(String role);
+
+    default void setContextIdentity(String userId) {
```
[the remark this answers is not in the document]
Comment (elguardian) [reply kie-internal/src/main/java/org/kie/internal/identity/IdentityProvider.java:31]: It is the identity of the context not the other way round.

File: kie-internal/src/main/java/org/kie/internal/identity/IdentityProvider.java [hunk 2]
```diff
@@ -27,4 +27,12 @@
...
+        // do nothing
+    }
+
+    default void removeContextIdentity() {
```
[the remark this answers is not in the document]
Comment (elguardian) [reply kie-internal/src/main/java/org/kie/internal/identity/IdentityProvider.java:35]: It is the identity of the context not the other way round.

**Review by afalhambra — APPROVED** [review]
Looking great, thanks @elguardian

**Pull request merged**